styles/roblox/catppuccin.user.css

```diff
@@ -50,17 +50,18 @@
     background-color: @base;
-      h1, h2, h3, h4, h5, .game-name-title, .dynamic-overflow-container .dynamic-ellipsis-item, .rbx-header .rbx-navbar li .nav-menu-title, .rbx-header .rbx-navbar-icon-group>li>button span[class^=rbx-text], #settings-popover-menu li a, .settings-text-field-container .settings-text-span-visible, .server-list-container-header .rbx-refresh, .text-lead, div, span, .game-creator .text-name, .text-name:visited, .profile-container .asset-item .item-name:hover, .text-emphasis, .main-header, .text, .radio label, .details-members-container .add-friends-option .add-friends-label, .text-name:link, .text-robux,  .table>tbody>tr>td, .table>tbody>tr>th, .table>tfoot>tr>td, .table>tfoot>tr>th, .table>thead>tr>td, .table>thead>tr>th, .icon-robux-price-container .text-robux-lg, .rbx-header .navbar-search .navbar-search-option .new-navbar-search-anchor .navbar-list-option-text{
+      h1, h2, h3, h4, h5, .game-name-title, .dynamic-overflow-container .dynamic-ellipsis-item, .rbx-header .rbx-navbar li .nav-menu-title, .rbx-header .rbx-navbar-icon-group>li>button span[class^=rbx-text], #settings-popover-menu li a, .settings-text-field-container .settings-text-span-visible, .server-list-container-header .rbx-refresh, .text-lead, div, span, .game-creator .text-name, .text-name:visited, .profile-container .asset-item .item-name:hover, .text-emphasis, .main-header, .text, .radio label, .details-members-container .add-friends-option .add-friends-label, .text-name:link, .text-robux,  .table>tbody>tr>td, .table>tbody>tr>th, .table>tfoot>tr>td, .table>tfoot>tr>th, .table>thead>tr>td, .table>thead>tr>th, .icon-robux-price-container .text-robux-lg, .rbx-header .navbar-search .navbar-search-option .new-navbar-search-anchor .navbar-list-option-text, #games-info-tooltip .tooltip-inner, .message-summary-content .subject, .paired-name span:first-child, .content .btr-currentPage input{ 
           color: @text;
       }
 
       .rbx-header .rbx-navbar li:hover, .border, .input-group-btn .input-dropdown-btn:hover, .input-group-btn .input-dropdown-btn:active, .input-group-btn .input-dropdown-btn:focus{
           border-color: @text;
       }
-      .btn-secondary-xs:link, .age-rating-age-bracket, .btn-secondary-xs:visited, .text-link, .select-option:hover, .text-footer-nav, .text-footer-nav:active, .text-footer-nav:link, .text-footer-nav:visited, .chat-friend-list .chat-friend-container .chat-friend-info .read, #serverFilters, .popover-content .dropdown-menu li a, .popover-content .dropdown-menu li button{
+      .btn-secondary-xs:link, .age-rating-age-bracket, .btn-secondary-xs:visited, .text-link, .select-option:hover, .text-footer-nav, .text-footer-nav:active, .text-footer-nav:link, .text-footer-nav:visited, .chat-friend-list .chat-friend-container .chat-friend-info .read, .popover-content .dropdown-menu li a, .popover-content .dropdown-menu li button{
           color: @text !important;
+          border-color: @text !important;
```

Comment:
Hmm this is a very strong border color. How does it look?

Reply:
I don't remember all the instances where such rules would apply but from my limited knowledge and remembrance I believe its replacing colors that were already full #fff white or near-white. So it ultimately looks the same between the unmodified (Roblox default) theme and Catppuccin theme.

Reply:
We don't want it to look the same necessarily? IRRC that applies to buttons and such right?

Reply:
I can't remember a situation where the ``border-color: @text !important;`` rule applies but here's a situation where the ``color: @text !important`` rule applies and I do believe that it does fit well within the theme and doesn't particularly stick out compared to the rest of the theme.

I'm unsure where the border rule applies since from what I can tell it applies in only a few places (the arrow icon looks different due to the aforementioned issues with replacing the .svg file) 
![image](https://github.com/catppuccin/userstyles/assets/70731807/816700a0-25f0-448d-ab7b-c3c533afc968)